cpp/src/codegen/node.h

```diff
@@ -38,6 +40,11 @@ class Node {
   /// Derived classes should simply invoke the Visit api of the visitor.
   virtual Status Accept(NodeVisitor &visitor) const = 0;
 
+  // for debugging purposes
```

Comment:
- we'll likely have good uses of this. pls remove the "for debugging".
- pls add a unit test